Tests/WebTest/SitemapTest.php

```diff
@@ -87,14 +90,19 @@ public function getFormats()
                          <xhtml:link href="http://localhost/sitemap-aware?_locale=de" hreflang="de" rel="alternate"/>
                      </url>
                      <url>
+                         <loc>http://localhost/sitemap-aware-last-mod-date</loc>
+                         <lastmod>2016-07-06T00:00:00+02:00</lastmod>
```

Comment:
That value is created in https://github.com/symfony-cmf/seo-bundle/pull/295/files#diff-c2db5dda6526964e483d707ab1118506R86 Same way we do it for the unit-tests. But the travis tests says `2016-07-05T22:00:00+00:00` same time but other timezone. @dbu you know what happens on phpcr-odm when saving DateTime?